release-schema.json

```
@@ -127,6 +144,16 @@
           "type": "string",
           "description": "A local identifier for this lot, such as a lot number. This is used in relatedLots references at the item, document and award level."
         },
+        "identifiers": {
+          "title": "Identifiers",
+          "description": "Identifiers for this lot. This field can be used to provide internal identifiers for the lot, such as identifiers from a buyer's document management system or procurement planning system.",
```

Why "procurement planning system"? I removed planning to avoid confusion between planning processes and contracting processes. Same change for LotGroup.

```suggestion
          "description": "Identifiers for this lot. This field can be used to provide internal identifiers for the lot, such as identifiers from a buyer's document management system or procurement system.",
```

---

That wording came from the eForms field description, but I agree it's better to remove 'planning'.

---

https://github.com/open-contracting-extensions/ocds_lots_extension/pull/41/commits/27ca5b619d07283942c1b2bd635ccaade36aaa8e updates `LotGroup`, although the commit message erroneously references `Lot`.